### ISSUE TITLE
Ability to enumerate registered hooks

### ISSUE THREAD

**Issue by TETYYS** [winsiderss/systeminformer#1947]
### Description of the feature, modification, idea or suggestion

I think someone asked about this before in old PH forum, but I would like to have it here again since I feel like there are more capabilities with SystemInformer.

There is no application that I know of that has ability to enumerate and display [global hooks](https://learn.microsoft.com/en-us/windows/win32/winmsg/about-hooks). I imagine the reason is that the structure where these hooks are stored is undocumented and probably in kernel memory somewhere, but [undocumented things don't stop SystemInformer](https://github.com/winsiderss/systeminformer/blob/master/KSystemInformer/dyndata.c), do they?

This capability would help to detect keyloggers or applications causing system instability. If one hooks [WH_MOUSE_LL](https://learn.microsoft.com/en-us/windows/win32/winmsg/about-hooks#wh_mouse_ll) and returns from callback function with a delay, user will feel that delay in his cursor movement.

**Comment by TETYYS** [comment]
also related https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerhotkey , but thats a completely different thing that would require a completely different implementation

**Comment by TETYYS** [comment]
Ok nevermind this is where I remember seeing it - https://github.com/tigros/HookTools. I'm just gonna leave the issue in case you want to consider it for integration into SystemInformer

**Comment by realyukii** [comment]
Hi, I came from the [HookTools readme](https://github.com/tigros/HookTools) that mention [ProcessHacker's source forge](http://processhacker.sourceforge.net/) but it redirect to the systeminformer, does this project support the HookTools plugin? I tried install it on plugins directory but not sure if it's the right way to integrate it, did you managed to get it work @TETYYS ?

**Comment by TETYYS** [comment]
No, I did not try it, I expect it to not work considering last commit was 8 years ago

**Comment by dmex** [comment]
<img width="123" height="576" alt="Image" src="https://github.com/user-attachments/assets/055cbaf7-886d-40d3-95b2-5f01072b1ed1" />

**Comment by TETYYS** [comment]
cool, is that new? I guess there is still some effort needed to resolve hook type

**Comment by diversenok** [comment]
That actually might be a challenge. The CSR shared heap stores the owner TID (which is really nice!), but the hook object itself doesn't seem to be present on the desktop heap

<img width="843" height="342" alt="Image" src="https://github.com/user-attachments/assets/d6f4d448-1aae-4cd9-8acb-fd3679fc083d" />
